src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -103,8 +110,8 @@ class BaseRequestHandler(RequestHandler):
 
       class TornadoInstance(BaseRequestHandler):
 
-        # Prefix of methods names
-        METHOD_PREFIX = "export_"
+        # Prefix of methods names if need to use a special prefix. By default its "export_".
```

Comment:
`export_` its by default. `web_` its default for `TornadoREST`, but this is not the first time this question has been raised, so I will try to avoid using `web_`. and do the appropriate PR